src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        int numberOfBalls = 3;
+        Ball[] balls = new Ball[numberOfBalls];
+        for (int i = 0; i < numberOfBalls; i++) {
+            balls[i] = lottery.getRandomBall();
+            System.out.println(balls[i]);
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```
no need for an array